### PR TITLE
[Release-v1.20.0] Changes for prepare release v1.20.0

### PR DESCRIPTION
**Pull request by sfc-gh-yzou**
1. Which Jira issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   
Update CHANGELOG, META.YAML and version.py

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
      - [ ] If this test skips Local Testing mode, I'm requesting review from @snowflakedb/local-testing
   - [ ] I am adding new logging messages
   - [ ] I am adding a new telemetry message
   - [ ] I am adding new credentials
   - [ ] I am adding a new dependency
   - [ ] If this is a new feature/behavior, I'm adding the Local Testing parity changes.

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,15 +9,13 @@
 - Added distributed tracing using open telemetry APIs for table stored procedure function in `DataFrame`:
   - _execute_and_get_query_id
 - Added support for function `arrays_zip`.
-- Allow `df.plot()` and `series.plot()` to be called, materializing the data into the local client
```
Comment (sfc-gh-aalam) [inline CHANGELOG.md:12]: why is this being removed?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,15 +9,13 @@
...
 - Improves performance for binary column expression and df._in by avoiding unnecessary cast for numeric values. This optimization can be enabled through session.eliminate_numeric_sql_value_cast_enabled = True.
 - Improved error message for `write_pandas` when target table does not exists and `auto_create_table=False`.
 - Added open telemetry tracing on UDxF functions in snowpark.
 - Added open telemetry tracing on stored procedure registration in snowpark.
 - Added a new optional parameter called `format_json` to the `Session.SessionBuilder.app_name` function that allows to set the app name in the `Session.query_tag` in JSON format. By default, this parameter is set to `False`.
 
 #### Bug Fixes
-- Fixed a bug regarding precision loss when converting to Snowpark pandas `DataFrame` or `Series` with `dtype=np.uint64`.
```
Comment (sfc-gh-yuwang) [inline CHANGELOG.md:20]: is this removed because of duplicate?

**Review by sfc-gh-mheavin — APPROVED** [review]
lgtm

File: CHANGELOG.md (hunk 2 above)
Comment (sfc-gh-yzou) [reply CHANGELOG.md:20]: Yes, and it is a snowpark pandas change, shouldn't be under snowpark python api

File: CHANGELOG.md (hunk 1 above)
Comment (sfc-gh-aalam) [reply CHANGELOG.md:12]: thanks.

**Review by sfc-gh-aalam — APPROVED** [review]

**Review by sfc-gh-yuwang — APPROVED** [review]

**Pull request closed without merging**